conductr_cli/sandbox_run_jvm.py

```diff
@@ -235,6 +235,22 @@ def validate_jvm_support():
         raise JavaCallError('Failure calling `java -version`')
 
 
+def validate_hostname_lookup():
+    """
+    Validates if the hostname lookup by Java is slow given the issue https://github.com/akka/akka/issues/22160
+    Validation fails if the hostname is not part of the /etc/hosts file and the OS is macOS.
+    A HostnameLookupError is raised if the validation fails
+    """
+    if host.is_macos():
```

Comment:
I do think we should check the Java version also as the CLI can operate on a machine where non ConductR dev occurs.